lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -621,6 +629,13 @@ def write(chunk)
           severity = compute_severity(
             entry_resource.type, record, entry_common_labels)
 
+          # Get trace resource for LogEntry "trace" field per specified config.
```

Comment:
+1. We should avoid resource as much as possible unless it has something specific to do with Stackdriver defined resources (i.e. monitored resources).